distributed/distributed.yaml

```diff
@@ -24,6 +24,7 @@ distributed:
     pickle: True            # Is the scheduler allowed to deserialize arbitrary bytestrings
     preload: []             # Run custom modules with Scheduler
     preload-argv: []        # See https://docs.dask.org/en/latest/setup/custom-startup.html
+    resolve-worker-hostname: true # Whether or not the worker hostname is resolved to an IP address at connection time
```

Comment:
What should the default be here?  Should we change this to false?

cc @jacobtomlinson

Reply:
Eventually, yes, I would argue this should become `False` given the potential benefit to TLS and dual-stack deploys.  Seems prudent to ease into it though - have a release or two with this set to `True`.

Reply:
Prudence?  Bah.

Reply:
I would argue that we should default to false. @mrocklin mentioned in #4070 that this is the result of being developed on a system with a poor DNS service. In general applications shouldn't resolve and cache DNS like this, so I think making that thge default would be good. But given that this has been an issue in the past it makes sense to leave a confgurable option to turn it on.

Reply:
Great, let's be aggressive here and change the default to false

Reply:
```suggestion
    resolve-worker-hostname: false # Whether or not the worker hostname is resolved to an IP address at connection time
```